eox_tenant/auth.py

```diff
@@ -0,0 +1,95 @@
+
+"""
+This file implements the authentication backend for the openedx platform.
+"""
+import logging
+import re
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.contrib.auth.backends import AllowAllUsersModelBackend as UserModelBackend
+
+from django.utils.translation import ugettext as _
+
+from ratelimitbackend.backends import RateLimitMixin
```

Comment:
would we need to add this dependency to eox-tenant dependencies? The inheritance didn't work the way we expected by using the `EdxRateLimitedAllowAllUsersModelBackend` class?

Reply:
Yes, with the current changes, we need to add that dependency.